code/public/index.html

```diff
@@ -13,13 +14,13 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>Technigo React App</title>
-  </head>
+  <title>Technigo React App</title>
```

Comment:
Remember to change this little thing to make the title reflect you project, shown in the browser tab for example.